src/components/GeneralInfo/EditButton/EditButtonWithAccess.test.js

```diff
@@ -29,24 +27,20 @@ describe('EditButton with access', () => {
   });
 
   it('enables with permission', () => {
-    const wrapper = mount(<EditButton onClick={onClick} link={link} />);
+    render(<EditButton onClick={onClick} link={link} />);
 
-    expect(wrapper.find(Tooltip)).toHaveLength(0);
-    expect(wrapper.find(PencilAltIcon)).toHaveLength(1);
-    expect(wrapper.find('a').props().href).toEqual(
+    expect(screen.getByRole('link', { name: /edit/i })).toBeVisible();
```

Comment:
Should this not query the Tooltip? ... or is that not possible anymore with the HTML rendered?

Reply:
Here the button is active and doesn't have any supportive tooltips. Though, I check the tooltips content at EditButtonWithNoAccess.test.js (the other neighbour test file).